utils/netlink.go

```diff
@@ -50,6 +50,34 @@ func DefaultNetMTU() (string, error) {
 	return fmt.Sprint(b.MTU), nil
 }
 
+func CheckBrInUse(brname string) (bool, error) {
+	InUse := false
+	l, err := netlink.LinkList()
+	if err != nil {
+		return InUse, err
+	}
+	mgmtbr, err := netlink.LinkByName(brname)
+	if err != nil {
+		return InUse, err
+	}
+	mgmtbridx := mgmtbr.Attrs().Index
+	for _, link := range l {
+		if link.Attrs().MasterIndex == mgmtbridx {
+			InUse = true
+			break
+		}
+	}
+	return InUse, nil
+}
+
+func DeleteNetworkInterface(name string) error {
```

Comment:
maybe this should be better called `DeleteLinkByName`?

Reply:
done